packages/telemetry/src/frcat-entrypoint.js

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env node
+/* global process, BigUint64Array */
+// frcat - print out the contents of a flight recorder
```

Comment:
maybe add a note that this only works correctly on stable/inactive recorder files, where the writing process has terminated